manifests/init.pp

```diff
@@ -121,7 +121,7 @@
 #
 class simp (
   Hash                            $scenario_map,
-  Enum['simp', 'simp_lite', 'poss', 'none'] $scenario         = 'simp',
+  String                          $scenario                   = 'simp',
```

Comment:
Either change this back, or update the documentation.

Without the Enum list, nobody has any idea what can actually go here.